packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -440,15 +553,6 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
     })
   }
 
-  private attachRegistrationListeners() {
-    this.registrationService.on(SocketActionTypes.OWNER_CERTIFICATE_ISSUED, payload => {
-      this.serverIoProvider.io.emit(SocketActionTypes.OWNER_CERTIFICATE_ISSUED, payload)
-    })
-    this.registrationService.on(RegistrationEvents.ERROR, payload => {
-      emitError(this.serverIoProvider.io, payload)
-    })
-  }
-
```

Comment:
Not needed anymore since we call to the registration service directly and await on its return